### ISSUE TITLE
Feature request: allow frecuent updates to the scummvm binary from master

### ISSUE THREAD

**Issue by neuromancer** [chkuendig/scummvm-demo#10]
I wonder how this repository can be updated from time to time, to allow users to play with a very fresh version of scummvm. It is easy to configure github actions to run once per day (which matches what scummvm servers are doing with the daily builds), however, right now you will need to manually rebase this PR in order to get the latest changes from master, so it seems there is no nice way to handle this, until this branch is merged.

On top of that, if a user loaded the scummvm demo before, when a new version is pushed, they will see the old version (and no indication of a new one). I guess the browser creates a persistent storage and won't overwrite it, so they need to clear the data and refresh. Ideally, it should be an easy way to allow the user to upgrade to the most recent version, showing an unobtrusive message or so.

**Comment by chkuendig** [comment]
You are right, there's no great way to handle this until the branch is merged. And even then, I would keep this repo probably to test against my branch. I imagine people working on this and needing a nightly to setup their own pages (or build locally), and eventually scummvm having an official web version (which would be versioned and updated accordingly)

**Comment by chkuendig** [comment]
I was rethinking this. I think it would be fairly straightforward to store the build version in localStorage and when that changes, re-run through the default init and refresh all games which were added or removed (or just overwrite the default ini again - but that would drop all settings)
